include/eve/function/gcd.hpp

```diff
@@ -43,20 +43,11 @@ namespace eve
   //!
   //!@warning
   //!    `p` and `n` can be of any [real values](@ref eve::value) type, but when the types are not integral
-  //!    the greatest common divisor is defined only if `p` and `n` element are [flint](@ref eve::is_flint). If it is not the
-  //!    case the corresponding result will be Nan.
+  //!    the greatest common divisor is defined only if `p` and `n` element are [flint](@ref eve::is_flint). If any is not flint
+  //!    the function will assert.
```

Comment:
assert is not a specification. Prefers:
"If any of the arguments is flint, then the behavior is undefined."